synapse/cortex.py

```diff
@@ -5854,13 +5854,12 @@ async def count(self, text, opts=None):
                 try:
                     return await proxy.count(text, opts=mirropts)
 
-                except s_exc.TimeOut:
+                except s_exc.TimeOut as e:
+                    extra.update(e.errinfo)
```

Comment:
The `extra` dictionary contains a nested synapse dictionary that we use.
```suggestion
                    extra['synapse'].update(e.errinfo)
```